modules/bene_promo_redirect/src/Form/PromoRedirectSettings.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Drupal\bene_promo_redirect\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Configure Custom Texts.
+ */
+class PromoRedirectSettings extends ConfigFormBase {
+
+  /**
+   * Returns a unique string identifying the form.
+   *
+   * @return string
+   *   The unique string identifying the form.
+   */
+  public function getFormId() {
+    return 'bene_promo_redirect_settings';
+  }
+
+  /**
+   * Gets the configuration names that will be editable.
+   *
+   * @return array
+   *   An array of configuration object names that are editable if called in
+   *   conjunction with the trait's config() method.
+   */
+  protected function getEditableConfigNames() {
+    return ['bene_promo_redirect.settings'];
+  }
+
+  /**
+   * Settings for "Promo Redirect" behavior.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The current state of the form.
+   *
+   * @return array
+   *   Form definition array.
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+
+    $form = parent::buildForm($form, $form_state);
+
+    // Add form header describing purpose and use of form.
+    $form['header'] = [
+      '#type' => 'markup',
+      '#markup' => t('<h3>Temporarily redirect your homepage, or all pages, to a promotional or campaign page within our without the site.</h3><p>Use carefully! Misconfiguration can result in frustration.</p>'),
```

Comment:
Typically translatable strings don't include markup, see https://www.drupal.org/docs/7/api/localization-api/dynamic-or-static-links-and-html-in-translatable-strings